src/components/NetworkListTable.tsx

```diff
@@ -3,35 +3,38 @@ import { useQuery } from "@tanstack/react-query";
 import { queryKeys } from "util/queryKeys";
 import { MainTable } from "@canonical/react-components";
 import Loader from "components/Loader";
-import { LxdInstance } from "types/instance";
 import { fetchNetworks } from "api/networks";
 import { isNicDevice } from "util/devices";
 import ResourceLink from "components/ResourceLink";
+import { useParams } from "react-router-dom";
+import { LxdDevices } from "types/device";
 
 interface Props {
-  instance: LxdInstance;
   onFailure: (title: string, e: unknown) => void;
+  networkDevices: LxdDevices;
 }
 
-const InstanceOverviewNetworks: FC<Props> = ({ instance, onFailure }) => {
+const NetworkListTable: FC<Props> = ({ onFailure, networkDevices }) => {
+  const { project } = useParams<{ project: string }>();
+
   const {
     data: networks = [],
     error,
     isLoading,
   } = useQuery({
-    queryKey: [queryKeys.projects, instance.project, queryKeys.networks],
-    queryFn: () => fetchNetworks(instance.project),
+    queryKey: [queryKeys.projects, project, queryKeys.networks],
+    queryFn: () => fetchNetworks(project as string),
   });
 
   if (error) {
     onFailure("Loading networks failed", error);
   }
 
-  const instanceNetworks = Object.values(instance.expanded_devices ?? {})
+  const overviewNetworks = Object.values(networkDevices ?? {})
```

Comment:
There is no concept of overview in this component. Yes, it is used in that context of overview pages. But only for now, and naming of variables should be optimal for the component it lives in. So probably better to call this just "networks" and drop the overview prefix.